dist/formats/specialist_document/frontend/schema.json

```diff
@@ -229,8 +229,7 @@
       "additionalProperties": false,
       "required": [
         "body",
-        "metadata",
-        "change_history"
```

Comment:
As said on slack, I'm unsure if this change to the frontend schema is intentional.